File: programmers/lv0/코딩기초트레이닝/연산/문자열섞기/Solution.go
package main

func solution(str1 string, str2 string) string {
	answer := ""
	for i := 0; i < len(str1); i++ {
		s1 := string(str1[i])
		s2 := string(str2[i])
		answer += s1 + s2
	}
	return answer
}

File: programmers/lv0/코딩기초트레이닝/연산/문자열섞기/Solution.swift
import Foundation

func solution(_ str1:String, _ str2:String) -> String {
//    return zip(str1, str2).map { String($0) + String($1) }.joined()
    var result = ""
    for i in 0..<str1.count {
        let idx = str1.index(str1.startIndex, offsetBy: i)

        result.append(str1[idx])
        result.append(str2[idx])
    }
    return result
}